charts/r2devops/templates/service.yaml

```
@@ -5,9 +5,16 @@
 apiVersion: v1
 kind: Service
 metadata:
+  annotations:
+    {{- range $key, $val := $.Values.additionalAnnotations }}
+    {{ $key }}: {{ $val | quote }}
+    {{- end }}
```

Avoid empty `annotation` key if no `additionalAnnotations`:

```suggestion
  {{- with $.Values.additionalAnnotations }}
  annotations:
    {{- range $key, $val := . }}
    {{ $key }}: {{ $val | quote }}
    {{- end }}
  {{- end }}
```